web_app/hazard-reporting-map/components/RegistrationForm.jsx

```diff
@@ -41,33 +41,49 @@ const RegistrationForm = () => {
             {error && <div className="text-2xl text-center text-red-500">{error}</div>}
             <form className="login-form" onSubmit={onSubmit}>
                 <div>
-                    <label htmlFor="fname">Full Name</label>
-                    <input type="text" name="fname" id="fname" />
+                    <label htmlFor="fname" className="text-gray-600">
```

Comment:
only Fullname , mobile and email field are in the register form now according to update